packages/cli/workspace-loader/src/types/Workspace.ts

```diff
@@ -36,7 +36,9 @@ export interface Spec {
 
 export interface SpecImportSettings {
     audiences: string[];
+    sdkLanguage: generatorsYml.GenerationLanguage | undefined;
     shouldUseTitleAsName: boolean;
+    shouldUseUndiscriminatedUnionsForDiscriminated: boolean;
```

Comment:
thoughts on a better name here? just brutal for a user to add and we actually need to roll this out broadly at vocode, vellum, all new demos etc